master/internal/authz/obfuscate.go

```diff
@@ -0,0 +1,86 @@
+package authz
```

Comment:
if you wanted to move https://github.com/determined-ai/determined/blob/93095ee0a2866be3372beacaa29877a481c8afa9/master/internal/api_job.go#L16 to here as well this would be a good time.